### PR TITLE
Create rocketchat.sls

### PR DESCRIPTION
**Pull request by airstream**

### CODE REVIEW

File: rocketchat.sls [hunk 1]
```diff
@@ -0,0 +1,14 @@
+rocketchat:
+{% for version in ['2.10.1',
+                   '2.9.0'] %}
+  '{{ version }}':
+    {% set PROGRAM_FILES = "%ProgramFiles%" %}
```
Comment (twangboy) [inline rocketchat.sls]: Put this at the top of the file (outside the for loop) so that it isn't defined during each iteration of the loop

**Comment by airstream** [comment]
Updated. Please push to master branch.

**Comment by airstream** [comment]
@TheBigBear @twangboy Sorry, but you did not add rocketchat.sls sls to salt-winrepo-ng master branch.

**Comment by TheBigBear** [comment]
@airstream thanks for this PR, merged as requested. sorry for the delay, life can get busy at times ...

**Pull request merged**